src/styles/page.css

```diff
@@ -265,6 +265,9 @@ img + em{
 /* ARTICLE SUMMARY VIEW */
 /*******************************/
 
+a:has(.article-summary){
```

Comment:
couldnt figure out how to add a class to the parent element of article-summary so using this workaround to avoid the summary descriptions having red underline for some reason